src/Microsoft.DotNet.XHarness.Apple/AppRunner.cs

```diff
@@ -219,7 +229,7 @@ public class AppRunner : AppRunnerBase
             {
                 await crashReporter.StartCaptureAsync();
 
-                _mainLog.WriteLine("Starting test run");
+                _mainLog.WriteLine("Starting the app");
```

Comment:
It is a test run, right? Why even change it?

Reply:
For the `app run` command, it's not a test run